packages/turbo/ios/RNVisitableView.swift

```diff
@@ -97,6 +102,7 @@ class RNVisitableView: UIView, RNSessionSubscriber {
   }
 
   public func refresh() {
+    controller.visitableURL = URL(string: String(url))
     session.visit(controller, action: .replace)
```

Comment:
Hm. I don't think we want to support refreshing if the controller is `nil`; we'd be unmounted, right?

Reply:
Let's add a guard instead and not do anything when the controller is not there.

Also let's make the `controller` optional and remove the `_controller` prop.